### PR TITLE
tidying css, renaming to correct classes

### PR DESCRIPTION
**Pull request by clmntnbrn**
I combined the CSS for several elements, which means we can remove classes. Also named elements correctly (eg `button` instead of `<div class="btn">`

### CODE REVIEW

**Comment by clmntnbrn** [comment]
I have no idea what just happened

@sashaward ready for review!

File: css/styles.css [hunk 1]
```diff
@@ -108,66 +113,78 @@ h1 {
...
-  margin-bottom: 7px;
+  margin-bottom: 0.5rem;
 }
 
-.field {
+select,
+input {
+  -webkit-border-radius: 0;
+  -moz-border-radius: 0;
+  border-radius: 0px !important;
```
Comment (sashaward) [inline css/styles.css]: WTF CLEM!

**Review by sashaward — CHANGES_REQUESTED** [review]

**Comment by clmntnbrn** [comment]
done MOFO @sashaward

**Review by sashaward — APPROVED** [review]

**Pull request merged**